### PR TITLE
Clockify ids

### PR DESCRIPTION
**Pull request by ThorpeJosh**

### CODE REVIEW

File: tp_timesheet/clockify_timesheet.py [hunk 1]
```diff
@@ -49,15 +49,17 @@ def submit_clockify(self, date, working_hours, dry_run=False):
             )
         logger.debug("POST:  %s\nResponse: %s", time_entry_json, response.text)
 
-    def get_workspace_id(self):
+    def get_workspace_user_id(self):
         """Send request to get workspace id"""
         get_request = requests.get(
             "https://api.clockify.me/api/v1/user",
```
Comment (ThorpeJosh) [inline tp_timesheet/clockify_timesheet.py:55]: The base api URL is available in `self.api_base_endpoint`

File: tp_timesheet/clockify_timesheet.py [hunk 2]
```diff
@@ -49,15 +49,17 @@ def submit_clockify(self, date, working_hours, dry_run=False):
...
             headers={"X-Api-Key": self.api_key},
             timeout=2,
         )
```
Comment (ThorpeJosh) [inline tp_timesheet/clockify_timesheet.py:58]: Can you please check response is 200 else raise a Value error with `response.text` included in message? Should help quickly identify issue if something goes wrong. 

Can check `submit_clockify()` above for example

**Comment by ThorpeJosh** [comment]
Sorry @siewyeng I merged your PR prematurely. Discovered a few issues when reviewing @lamchuntat's PR. Can you please make the above changes then we will merge again?

**Comment by siewyeng** [comment]
@ThorpeJosh yes I can!

**Pull request merged**